nab/detectors/numenta/numenta_detector.py

```diff
@@ -66,28 +68,18 @@ def handleRecord(self, inputData):
 
 
   def initialize(self):
-    calcRange = abs(self.inputMax - self.inputMin)
-    calcPad = calcRange * .2
+    # Get config params, setting the RDSE resolution
+    rangePadding = abs(self.inputMax - self.inputMin) * 0.2
```

Comment:
@subutai I tried several different values for the range padding and found 0.2 to give the best results.